src/DssSpell.t.sol

```diff
@@ -1088,23 +1091,23 @@ contract DssSpellTest is DSTest, DSMath {
         // Track Median authorizations here
 
         address SET_AAVE    = 0x8b1C079f8192706532cC0Bf0C02dcC4fF40d045D;
```

Comment:
Why are we still running this test in this spell?

Reply:
I think we decided to leave this so we begin to capture the config.

Reply:
sorry but what do you mean? I think I missed that discussion.